.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,4 @@
-**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in [The Power of Us Hub](https://powerofus.force.com).
-For feature requests, please use the Ideas tab in [The Power of Us Hub](https://powerofus.force.com).**
+**Note: FOR INTERNAL USE ONLY. For support questions, please use the K-12 [Sector: K-12 Education group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi7SAE?tab=discussion).
```

Comment:
@mrrigney One small edit:
...please use the [Sector: K-12 Education group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi7SAE) in Trailblazer Community.

Reply:
@mrrigney Sorry if this is a "Level 1 Error" but I have refreshed and I'm still seeing a problem in line 1. It says: "For support questions, please use the **K-12** [Sector: K-12 Education group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi7SAE?tab=discussion).

Reply:
@aarsvoboda try looking at this under the files changed tab, you should see
![image](https://user-images.githubusercontent.com/3503219/195637788-ebde4c71-eee9-4d30-8891-c56989d055af.png)